DigitalLearningSolutions.Web/Scripts/trackingSystem/centreCourseSetup.ts

```diff
@@ -0,0 +1,54 @@
+const copyCourseLinkClass = 'copy-course-link';
+const copyLinkIdPrefix = 'copy-course-';
+const launchCourseButtonIdPrefix = 'launch-course-';
+
+setupCourseLinkClipboardCopiers();
+
+function setupCourseLinkClipboardCopiers() {
+  const copyCourseLinks = document.getElementsByClassName(copyCourseLinkClass);
+
+  for (let i = 0; i < copyCourseLinks.length; i += 1) {

```

Comment:
Minor, but it would be much nicer to use JS forEach here (might have to do Array.from(..)) first like elsewhere where we deal with the elements.